policy-engine/src/openapi.rs

```diff
@@ -1,9 +1,23 @@
 use actix_web::{HttpRequest, HttpResponse};
-use openapiv3::OpenAPI;
+use openapiv3::{OpenAPI, ReferenceOr};
+use std::collections::HashSet;
 use AppState;
 
+/// Template for policy-engine OpenAPIv3 document.
 const SPEC: &str = include_str!("openapiv3.json");
 
+/// Template for an OpenAPIv3 query parameter.
+static PARAM_TEMPLATE: &str = r#"
```

Comment:
Asking for education since we now have both: please elaborate about using `static` over `const` for global variables since?

Reply:
Also there's only one reference to this in one function, so you could move it there.

Reply:
The difference is explained here: https://doc.rust-lang.org/book/ch19-01-unsafe-rust.html#accessing-or-modifying-a-mutable-static-variable.

I initially had it inline, but I thought it was adding noise when reading through the function, so moved it up.